src/content/code/plugin-api/layers/selected-layer.ts

```
@@ -0,0 +1,20 @@
+
+//1. Check if there is a selected layer and log its details
+if (reearth.layers.selected) {
+  console.log("Selected Layer ID:", reearth.layers.selected.id);
+  console.log("Selected Layer Title:", reearth.layers.selected.layer.title);
+} else {
+  console.log("No layer is currently selected.");
+}
+
+//2. Example function to update the visibility of the selected layer
+function toggleVisibilityOfSelectedLayer() {
+  if (reearth.layers.selected && reearth.layers.selected.layer.visible) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)